test/test_modules.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from pathlib import Path
+
+import yt_dlp_ejs.yt.solver
+
+CORE_PATH = Path('yt_dlp_ejs/yt/solver/core.min.js')
```

Comment:
This is using a relative directory. Maybe it would be better to do sth like this:
```py
BASE_PATH = Path(__file__).parent.parent
CORE_PATH = BASE_PATH / ...
```